_14-ex.Rmd

```diff
@@ -13,8 +13,8 @@ library(spDataLarge)
 
 E1. Download the csv file containing inhabitant information for a 100 m cell resolution (https://www.zensus2011.de/SharedDocs/Downloads/DE/Pressemitteilung/DemografischeGrunddaten/csv_Bevoelkerung_100m_Gitter.zip?__blob=publicationFile&v=3).
 Please note that the unzipped file has a size of 1.23 GB.
-To read it into R you can use `readr::read_csv`.
-This takes 30 seconds on a machine with 16 GB RAM.
+To read it into R, you can use `readr::read_csv`.
+This takes 30 seconds on a machine with 16-GB RAM.
```

Comment:
16-GB looks fine, not sure if we're doing that throughout. If not I tentatively suggest not making that change. Minor niggle.

Reply:
you are right, I'll change back